pkg/controller/scyllacluster/resource/resource_test.go

```diff
@@ -314,3 +321,590 @@ func TestMemberService(t *testing.T) {
 		})
 	}
 }
+
+func TestStatefulSetForRack(t *testing.T) {
+	basicRack := scyllav1.RackSpec{
```

Comment:
let's not edit a global variable, assignments are not deep copies

Reply:
changed to func newBasicRack